src/react/hooks.ts

```diff
@@ -246,3 +247,20 @@ export function useActionEffect(configId: string, effect: () => void) {
     return client.config.registerEffect(configId, effectRef.current);
   }, [client, configId, effect]);
 }
+
+/**
+ * React hook for accessing plugin style with live updates
+ * @returns {PluginStyle | undefined} Style properties from the workbook if available
+ */
+export function usePluginStyle(): PluginStyle | undefined {
+  const client = usePlugin();
+  const [style, setStyle] = useState<PluginStyle | undefined>(undefined);
+
+  useEffect(() => {
+    // Request initial style data on mount and subscribe to updates
+    void client.style.getStyle().then(setStyle);
```

Comment:
```suggestion
    void client.style.getStyle().then(response => setStyle(response));
```
This implementation is slick, but it's somewhat tricky to read